Mail/Views/Search/SearchViewModel.swift

```diff
@@ -182,8 +186,9 @@ enum SearchState {
 
         isLoading = true
         await tryOrDisplayError {
+            let searchFolderCopy = searchFolder.detached()
```

Comment:
Since searchFolder is already frozen we don't need to detach it

Reply:
I did that, and further renamed parameters where it made sense to explicit frozen state. see 59241b.